package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@playkit-js/moderation",
-  "version": "3.2.17",
+  "version": "3.2.19",
```

Comment:
version sets by `prod` script in github actions, so to avoid conflicts we shouldn't modify value manually